src/Runtime/EscaperRuntime.php

```diff
@@ -17,6 +17,7 @@
 
 final class EscaperRuntime implements RuntimeExtensionInterface
 {
+    /** @var array<callable(string $string, string $charset): string> */
```

Comment:
```suggestion
    /** @var array<string, callable(string $string, string $charset): string> */
```

This way, phpstan will be aware of the type of array keys